src/bosh-director/lib/bosh/blobstore_client/azurestoragecli_blobstore_client.rb

```diff
@@ -40,11 +40,8 @@ def initialize(options)
     def redacted_credential_properties_list
       %w[account_key]
     end
```

Comment:
Empty line between methods

Reply:
Hi @Malsourie,
thx for the hint. I changed it.